Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java

```diff
@@ -611,11 +613,22 @@ public void testFindAllIDs() throws DataException {
 
         allIDs = ServiceManager.getProcessService().findAllIDs(0L, 5);
         Assert.assertEquals("Wrong amount of id's in index", 5, allIDs.size());
-        Assert.assertTrue("id's contain wrong entries", allIDs.containsAll(Arrays.asList(5, 3, 1, 2, 6)));
+        Assert.assertEquals("Duplicate ids in index", allIDs.size(), new HashSet<Integer>(allIDs).size());
+        Integer maxId = allIDs.stream().mapToInt(Integer::intValue).max().getAsInt();
+        Integer minId = allIDs.stream().mapToInt(Integer::intValue).min().getAsInt();
+        Assert.assertTrue("Ids should all be smaller than 8", maxId < 8);
+        Assert.assertTrue("Ids should all be larger than 0", minId > 0);
+        
+        // This test is very strange. The method findAllIDs doesn't specify an order in which processes are supposed 
+        // to be returned. It could be any order ElasticSearch chooses. It's weird that this doesn't fail randomly.
+        // Assert.assertTrue("id's contain wrong entries", allIDs.containsAll(Arrays.asList(5, 3, 1, 2, 6)));
```

Comment:
Right, I would also suggest to delete this test because the test in `line:612` checks if allIDs contains wrong entries or not.
`Assert.assertTrue("id's contain wrong entries", allIDs.containsAll(Arrays.asList(5, 2, 6, 4, 1, 7, 3)));`

Reply:
Thank you for your feedback. I deleted both tests.